dev/src/main/java/org/eclipse/codewind/intellij/core/filetypes/CWSettingsFileTypeDetector.java

```diff
@@ -0,0 +1,54 @@
+/*******************************************************************************
+ * Copyright (c) 2020 IBM Corporation and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v2.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v20.html
+ *
+ * Contributors:
+ *     IBM Corporation - initial API and implementation
+ *******************************************************************************/
+package org.eclipse.codewind.intellij.core.filetypes;
+
+import com.intellij.openapi.fileTypes.ExactFileNameMatcher;
+import com.intellij.openapi.fileTypes.FileType;
+import com.intellij.openapi.fileTypes.FileTypeRegistry;
+import com.intellij.openapi.fileTypes.impl.FileTypeManagerImpl;
+import com.intellij.openapi.util.io.ByteSequence;
+import com.intellij.openapi.vfs.VirtualFile;
+import org.eclipse.codewind.intellij.core.constants.CoreConstants;
+import org.jetbrains.annotations.NotNull;
+import org.jetbrains.annotations.Nullable;
+
+import java.util.Collection;
+import java.util.Collections;
+
+public class CWSettingsFileTypeDetector implements FileTypeRegistry.FileTypeDetector {
+
+    public CWSettingsFileTypeDetector() {
+        FileTypeManagerImpl.getInstance().associate(CWSettingsFileType.INSTANCE, new ExactFileNameMatcher(CoreConstants.SETTINGS_FILE));
```

Comment:
Use `FileTypeManager.getInstance()` instead of `FileTypeManagerImpl.getInstance()`